src/struct_info.json

```diff
@@ -1857,14 +1873,14 @@
                 "nextInChain",
                 "label",
                 "layout",
-                "bindingCount",
```

Comment:
Unless I'm mistaken, I don't think it will work like this. Those definitions here determine the offsets applied in the JS wrapper to get to the underlying values in the C/C++ struct.
The new fields have been added *after* the deprecated fields, so by removing those from this mapping, the new offsets of `entryCount` and `entries` JS side will actually map to the offsets of the old fields `bindingCount` and `bindings` on the C struct, which are expected to not have been set, since we're expecting the non-deprecated usage here.
The JS value of those fields will therefore be garbage (again, unless I missed something).
It's kind of a stroke of bad luck, since if the new fields had been added *before* the deprecated fields, it would "magically" work (but I wouldn't like to be counting of that either and it could mess with the offsets of the fields right after if there are any).